examples/07_automatic_preprocessing_with_the_supervectorizer.py

```diff
@@ -0,0 +1,126 @@
+"""
+Automatic pre-processing with the SuperVectorizer
+=================================================
+
+In this notebook, we will illustrate the use of the `SuperVectorizer`, which
+can automatically classify columns of a dataset based on their type, and apply
+transformers based on this classification.
+To demonstrate that, we will use the `employee salaries` dataset.
```

Comment:
```suggestion
In this notebook, we illustrate the `SuperVectorizer`, which automatically
applies transformers to the different columns of a dataset.
We demonstrate it onthe `employee salaries` dataset.
```

Less words are better (and future tense is not needed)